The following values were not passed to `accelerate launch` and had defaults used instead:
	`--num_processes` was set to a value of `1`
	`--num_machines` was set to a value of `1`
	`--mixed_precision` was set to a value of `'no'`
	`--dynamo_backend` was set to a value of `'no'`
To avoid this warning pass in values for each of the problematic parameters or run `accelerate config`.
/net/tscratch/people/plgas2000/.conda/envs/env_zzsn/lib/python3.9/site-packages/diffusers/utils/outputs.py:63: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  torch.utils._pytree._register_pytree_node(
05/27/2024 21:51:20 - INFO - __main__ - Distributed environment: NO
Num processes: 1
Process index: 0
Local process index: 0
Device: cuda

Mixed precision type: fp16

You are using a model of type clip_text_model to instantiate a model of type . This is not supported for all configurations of models and can yield errors.
You are using a model of type clip_text_model to instantiate a model of type . This is not supported for all configurations of models and can yield errors.
/net/tscratch/people/plgas2000/.conda/envs/env_zzsn/lib/python3.9/site-packages/diffusers/utils/outputs.py:63: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  torch.utils._pytree._register_pytree_node(
{'clip_sample_range', 'variance_type', 'dynamic_thresholding_ratio', 'thresholding'} was not found in config. Values will be initialized to default values.
{'attention_type', 'dropout', 'reverse_transformer_layers_per_block'} was not found in config. Values will be initialized to default values.
Traceback (most recent call last):
  File "/net/tscratch/people/plgas2000/zzsn-project/scripts/train_ziplora/train_dreambooth_lora_sdxl.py", line 2013, in <module>
    main(args)
  File "/net/tscratch/people/plgas2000/zzsn-project/scripts/train_ziplora/train_dreambooth_lora_sdxl.py", line 1123, in main
    attn_module.to_q.set_lora_layer(
  File "/net/tscratch/people/plgas2000/.conda/envs/env_zzsn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1688, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'Linear' object has no attribute 'set_lora_layer'
Traceback (most recent call last):
  File "/net/tscratch/people/plgas2000/.conda/envs/env_zzsn/bin/accelerate", line 8, in <module>
    sys.exit(main())
  File "/net/tscratch/people/plgas2000/.conda/envs/env_zzsn/lib/python3.9/site-packages/accelerate/commands/accelerate_cli.py", line 46, in main
    args.func(args)
  File "/net/tscratch/people/plgas2000/.conda/envs/env_zzsn/lib/python3.9/site-packages/accelerate/commands/launch.py", line 1075, in launch_command
    simple_launcher(args)
  File "/net/tscratch/people/plgas2000/.conda/envs/env_zzsn/lib/python3.9/site-packages/accelerate/commands/launch.py", line 681, in simple_launcher
    raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)
subprocess.CalledProcessError: Command '['/net/tscratch/people/plgas2000/.conda/envs/env_zzsn/bin/python', 'scripts/train_ziplora/train_dreambooth_lora_sdxl.py', '--pretrained_model_name_or_path=stabilityai/stable-diffusion-xl-base-1.0', '--instance_data_dir=./dataset/styles/style1-cartoon', '--output_dir=./models/dreambooth_lora_sdxl_style1_cartoon_2024-05-27_21-51-07', '--instance_prompt=a cat of in crt style', '--rank=64', '--resolution=1024', '--train_batch_size=1', '--learning_rate=5e-5', '--report_to=wandb', '--lr_scheduler=constant', '--lr_warmup_steps=0', '--max_train_steps=1000', '--validation_prompt=a man in crt style', '--validation_epochs=50', '--seed=0', '--mixed_precision=fp16', '--enable_xformers_memory_efficient_attention', '--gradient_checkpointing', '--use_8bit_adam']' returned non-zero exit status 1.
